docs/index.md

```diff
@@ -2,15 +2,18 @@
 :maxdepth: 2
 :titlesonly:
 :caption: General
+:hidden:
 
-
-Overview <self>
 Readme <misc/readme_link>
 User Guide <userguide/userguide>
 Examples <examples/examples>
 How To Contribute <misc/contributing_link>
```

Comment:
Can we please shorten this to simply `Contribute`, we should keep the side tab as short as possible and no meaningful understanding is lost by this shortening